dev-util/mh/mh-1.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Generate help for Makefile"
+HOMEPAGE="https://github.com/oz123/mh"
+SRC_URI="https://github.com/oz123/mh/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+IUSE="test"
+KEYWORDS="~amd64"
```

Comment:
metadata block needs cleaning up

Reply:
Can you be specific? What should be removed?

Reply:
@oz123 I mistook this for another request to add a package to `::gentoo`. In general, we don't accept new packages to `::gentoo` anymore that are purely proxy maintained, for a number of reasons. Why not add it to `::guru`?

Reply:
I can do that. But I don't contribute the guru project. That means I will need to go throuh the process of gaining access before this gets useful.
Quite a bummer. Out of scope here. But the. Direct engagement with Gentoo developers is really cool. Having another layer of abstraction is perhaps reducing the load on Gentoo developers but it's making it harder to recruit new developers.